reqresp/README.md

```diff
@@ -0,0 +1,73 @@
+# Request/Response Sample
+
+This sample demonstrates how to send a request and get a response from a Temporal workflow. Two approaches to providing
+the caller a response are in the example:
+
+* Query poll - Caller can run queries polling for response (the slow/inefficient way)
+* Response activity - Caller can provide activity information on where to send the response to (the more robust way)
+
```

Comment:
I think _more robust_ is not really correct. Polling is robust. It just adds latency and can be less efficient.

Reply:
Will update

Reply:
Updated